kore/src/Kore/Internal/TermLike.hs

```diff
@@ -458,11 +469,31 @@ instance NFData variable => NFData (TermLike variable) where
 instance SortedVariable variable => Unparse (TermLike variable) where
     unparse term =
         case Recursive.project freshVarTerm of
-          (_ :< pat) -> unparse pat
+          (attrs :< pat) ->
+              Pretty.vsep
+                  $ catMaybes
+                    [ attrs Lens.^? getCallStackHead
+                    , Just $ unparse pat
+                    ]
       where
         freshVarTerm =
             externalizeFreshVariables
             $ mapVariables toVariable term
+        showCreated (name, loc) =
+            Pretty.hsep
+                [ "// Created by"
```

Comment:
I think we should prefer to use the `/* C-style */` comments because the pretty-printer can split and combine lines, which might break this comment.